modules/howtos/pages/encrypting-using-sdk.adoc

```diff
@@ -1,15 +1,173 @@
-= Field Level Encryption from the SDK
-:description: The Field Level Encryption library enables encryption and decryption of JSON fields, to support FIPS-140-2 compliance.
+= Field Level Encryption from the Node.js SDK
+:description:  A practical guide for getting started with Field-Level Encryption, showing how to encrypt and decrypt JSON fields using the Node.js SDK.
 :page-topic-type: howto
 :page-edition: Enterprise Edition
-:page-status: Developer Preview
-// // :page-aliases: ROOT:encrypting-using-sdk.adoc
+:page-aliases: ROOT:encrypting-using-sdk.adoc
 
 [abstract]
-{description}
+{description}
 
+For a high-level overview of this feature, see our xref:concept-docs:encryption.adoc[Encryption] page.
 
-Client-side implementation of Field Level Encryption is available in the previous version of the SDK.
-It will be enabled in a future release of the third generation SDK.
+[#package]

```

Comment:
I'm not sure where this modifying anchors started in the docs - but do we have many deep links that would break if we were to cease the practice now, and remove the burden of maintenance?

Reply:
As discussed on slack, seems like the anchors are redundant here, I'll remove these!